services/src/openkilda-gui/docker-compose.yml

```diff
@@ -3,4 +3,4 @@ services:
   openkilda_gui:
```

Comment:
Do you still need separate docker-compose.yml?

Reply:
No need, i believe.
removing